bundles/binding/org.openhab.binding.tcp/src/main/java/org/openhab/binding/tcp/protocol/internal/ProtocolGenericBindingProvider.java

```diff
@@ -190,13 +199,19 @@ private void parseBindingConfig(ProtocolBindingConfig config, Item item, String
      */
     private Command createCommandFromString(Item item, String commandAsString) throws BindingConfigParseException {
 
-        Command command = TypeParser.parseCommand(item.getAcceptedCommandTypes(), commandAsString);
+        if (CHANGED_COMMAND_KEY.equals(commandAsString)) {
+            return CHANGED_COMMAND_KEY;
+        } else if (WILDCARD_COMMAND_KEY.equals(commandAsString)) {
+            return WILDCARD_COMMAND_KEY;
+        } else {
+            Command command = TypeParser.parseCommand(item.getAcceptedCommandTypes(), commandAsString);
 
-        if (command == null) {
-            throw new BindingConfigParseException("couldn't create Command from '" + commandAsString + "' ");
-        }
+            if (command == null) {
+                throw new BindingConfigParseException("couldn't create Command from '" + commandAsString + "' ");
+            }
```

Comment:
I'm using the built in Github editor, but I extraced the code from the Exec Binding, so it should meet the owners code formatting standards.